libethash-cuda/ethash_cuda_miner.cpp

```diff
@@ -63,17 +63,12 @@ ethash_cuda_miner::search_hook::~search_hook() {}
 
 ethash_cuda_miner::ethash_cuda_miner()
 {
-	int devicesCount = getNumDevices();
-	m_light = new hash64_t*[devicesCount];
-	for(int i = 0; i < devicesCount; i++)
+	int deviceCount = getNumDevices();
+	m_light = std::vector<hash64_t*>(deviceCount);
```

Comment:
This is fine, but because `m_light` is a vector already, you can alternatively write it like `m_light.resize(deviceCount)`. Still it will init new elements with `nullptr`.